lib/ace/http2/service.ex

```diff
@@ -2,10 +2,9 @@ defmodule Ace.HTTP2.Service do
   @moduledoc """
   Run a supervised tree of HTTP/2.0 servers, all available on a single port.
 
-  An behaviour of an `Ace.HTTP2.Service` is defined by the application it runs.
-  The definition of an application is a behaviour module and argument list that will be used to start the worker.
+  A behaviour of an `Ace.HTTP2.Service` is defined by the application it runs.
```

Comment:
The task performed by an `Ace.HTTP2.Service` is a result of the worker processes run for each clients request.
a Module that exports start_link and its arguments are required so `Ace.HTTP2.Service` can start new workers